librecorps.md

```diff
@@ -0,0 +1,25 @@
+---
+short: LibreCorps
+title: LibreCorps
+slug: librecorps
+date: 2018-11-28
```

Comment:
Typically dates aren't put on non-posts in Jekyll. If you're looking for a "last changed" date, I would recommend a custom variable as "date" has a special meaning in Jekyll.

Reply:
@ct-martin Ah. It existed on some of the other pages so I followed example. I'll push a new commit to clean this up now.